src/sssom_schema/schema/sssom_schema.yaml

```
@@ -564,6 +573,9 @@ slots:
         description: "The RDFS label property (rdfs:label) was used to match the object."
       - value: skos:prefLabel
         description: "The SKOS preferred label property (skos:prefLabel) was used to match the object."
+    see_also:
+        - https://mapping-commons.github.io/sssom/mapping-justifications/#lexical-matching
+        - https://github.com/mapping-commons/sssom/issues/413
```

```suggestion
    see_also:
      - https://mapping-commons.github.io/sssom/mapping-justifications/#lexical-matching
      - https://github.com/mapping-commons/sssom/issues/413
```
Remove the extra level of indentation.